components/gitpod-protocol/go/gitpod-service.go

```diff
@@ -1756,6 +1756,7 @@ type PortConfig struct {
 	Visibility  string  `json:"visibility,omitempty"`
 	Description string  `json:"description,omitempty"`
 	Name        string  `json:"name,omitempty"`
+	Sort        uint32  `json:"sort,omitempty"`
```

Comment:
How does this work? Sort is normally the ordering - Ascending or Descending. How does a uint work here?

Reply:
`Sort` means the number for sort here, add it to avoid a lot of changes in supervisor